app/src/main/res/values-de/strings.xml

```diff
@@ -10,14 +10,14 @@
     <string name="camera_switch_error">Kamerawechsel fehlgeschlagen</string>
     <string name="click_to_resume_preview">Klicke auf das Bild, um bei der Vorschau zu bleiben</string>
     <string name="photo_not_saved">Das Foto konnte nicht gespeichert werden</string>
-    <string name="setting_resolution_failed">Setting proper resolution failed</string>
+    <string name="setting_resolution_failed">Korrekte Auflösung konnte nicht eingestellt werden</string>
 
     <!-- Other aspect ratio -->
     <string name="other">andere</string>
 
     <!-- FAQ -->
-    <string name="faq_1_title">What photo compression quality should I set?</string>
-    <string name="faq_1_text">It depends on your goal. For generic purposes most people advise using 75%-80%, when the image is still really good quality, but the file size is reduced drastically compared to 100%.</string>
+    <string name="faq_1_title">Was für eine Fotokomprimierungsqualität soll ich einstellen?</string>
+    <string name="faq_1_text">Das hängt von deinem Ziel ab. Für allgemeine Zwecke empfehlen die meisten 75  - 80 %, wodurch die Qualität immer noch sehr gut bleibt, aber die Dateigrösse drastisch reduziert ist im Vergleich zu 100 %.</string>
```

Comment:
what was your intention at "75  - 80 %"? Maybe you missed a percent character, or typed 2 spaces by mistake?

Reply:
It was indeed one space too much.
Thanks for correcting that.